CHANGES.rst

```diff
@@ -1,15 +1,219 @@
+  - Moving Carnaval / Mardi Gras / Fat Tuesday calculation into the `workalendar.core` module, because it's used in at least 3 countries and some States / Counties in the USA.
+
+
+v10.0.0 (unreleased)
+--------------------
+
+Incorporate changes from workalendar v10.0.0 (2020-06-05)
+
+- **BREAKING CHANGE**: the ``IsoRegistry.get_calendar_class()`` method has been removed from the code and should no longer be used (#375, #495).
+
+v9.2.0 (2020-06-02)
+
+- New Calendars
+
+- Added rules for all Switzerland Cantons, branching off the initial work by @brutasse (#497).
+
+v9.0.1 (2020-05-22)
+
+- Making the Israel calendar more efficient (#498).
+- Fixing duplicated holidays in Hong-Kong and Hong-Kong Bank holiday calendars (#496).
+- Integrating Hong-Kong holidays for 2021 (#496).
+
+v9.0.0 (unreleased)
+-------------------
+
+Incorporate changes from workalendar v9.0.0 (2020-04-24)
+
+- **BREAKING CHANGE**: the ``IsoRegistry.items()`` method has been removed from the API. You must use the ``get_calendars()`` to perform the same registry queries (#375, #491).
+- *Deprecation notice*: The usage of ``IsoRegistry.get_calendar_class()`` is strongly discouraged, in favor of ``get()``. The ``get_calendar_class`` method will be dropped in a further release. In the meantime, they'll be both equivalent (#375, #418).
+
+v8.4.0 (2020-04-17)
+
+- New Calendar
+
+  - Added Kenyan calendar, by @KidkArolis (#484)
+
+- Minor fixes
+
+  - Fixed Lithuania calendar to use the core flags for Assumption and All Saints (#468).
+  - Fixed Malta calendar ; January 1st was already included, no need to add it to the ``FIXED_HOLIDAYS`` property (#469).
+  - Small refactor in Netherlands calendar to use core constants (#470).
+
+v8.3.0 (2020-04-14)
+
+- Fixing Hong-Kong calendar, where SAT are common working days (#477).
+- Introducing Hong-Kong Bank calendar. For banks, Saturdays are non-working days (#477).
+
+v8.2.2 (2020-04-10)
+
+- Fixed Argentina's "Malvinas Day" date for 2020, shifted to March 31st because of the coronavirus crisis (#476).
+- Fixed Argentina's label for "Malvinas Day" and "Día de la Memoria" (#476).
+
+v8.2.1 (2020-04-03)
+
+- Added BrazilBankCalendar to support `include_` flags and make it possible to extend and change these flags to support custom bank calendars (#474).
+
+v8.2.0 (2020-03-13)
+
+- Added Belarus calendar, by @alexdoesstuff (#472).
+
+v8.1.0 (2020-02-07)
+
+- Added Israel holidays eves and removed holidays which are not affecting the working days in Israel (#461).
+- Fix warning in China's holidays to dynamically read supported years, thx @fredrike (#459).
+
+v8.0.2 (2020-01-24)
+
+- Fix several miscalculations in Georgia (USA) calendar (#451).
+
+v8.0.1 (2020-01-24)
 
-Fix version inference when installed from sdist.
+- Fix Family Day for British Columbia (Canada) which was switched from 2nd to 3rd Monday of February in 2019 - thx @jbroudou for the bug report (#454).
 
-v6.1.0
+v8.0.0
```

Comment:
Calendra follows its own versioning scheme, so since 6.1.0 was the last release, probably 7.0.0 will be the next release, and it will incorporate all of the changes from workalendra 8.0.1-14.1.

Reply:
I'm easy either way, but I can see some small advantage in keeping comparable numbers. Again, feel free to revise them if you like, or I'm happy to comply too.

Reply:
I looked at the file, and am concerned that having two sets of version numbers will be even more confusing to read. At least the current approach avoids that...

Reply:
The problem I've had in the past was that if calendra wanted to make a change independent of workalendar, it would cause an increment in version numbers. To honor semver, that increment might be a major (1.x) bump. So then calendra is constrained to only commit changes at the same scope as (and coordinated with) workalendar changes or its version numbers must get ahead of workalendar. In short, there's no way to honor semver and keep the numbers in sync. That's why I decided to honor semver and use an otherwise independent versioning scheme.

I use the "Incorporate changes from workalendar vN.N.N" in the changelog to provide a clear and direct association of changes inherited from the upstream. Let's keep that scheme for now.

Reply:
ok, i'll work this in.